src/lcm-arrowhead.py

```diff
@@ -1,30 +1,40 @@
 #!/usr/bin/env python3
 import lcm
 from robot import action_t
-import time
 from flask import Flask
 
 app = Flask(__name__)
 
 # Publish a action through lcm
-def publish_action(action):
+def publish_action(action, position):
     a = action_t()
     a.action = action
+    a.pos = position
     lcm.LCM().publish("ACTION", a.encode())
```

Comment:
Can you rebase on latest master and modify the existing arrowhead message type instead?

Reply:
This isn't resolved yet: modify and use the `io_arrowhead_t` struct on the channel `IO_ARROWHEAD` instead.